tests/cross/src/test/scala/tests/pc/PcRenameSuite.scala

```diff
@@ -488,4 +488,28 @@ class PcRenameSuite extends BasePcRenameSuite {
        |  } yield b
        |""".stripMargin,
   )
+
+  check(
+    "end-marker".tag(IgnoreScala2),
+    """|def <<he@@llo>>(a: Int) =
+       |  ???
+       |end <<hello>>
+       |""".stripMargin,
+  )
+
+  check(
+    "end-marker-with-comment".tag(IgnoreScala2),
+    """|def <<he@@llo>>(a: Int) =
+       |  ???
+       |end /* a comment */ <<hello>> /* a comment */
```

Comment:
```suggestion
       |end /* a comment */ <<hello>> // a comment
```
Nit, to test both types of comments. :slightly_smiling_face:

Reply:
Doesn't matter here, that comment is outside of definition span anyway.